index.bs

```diff
@@ -1684,6 +1692,12 @@ during registration.
         [=attestation statement=], as well as to decode and validate the [=authenticator data=] along with the
         [=JSON-serialized client data=]. For more details, see [[#sctn-attestation]], [[#generating-an-attestation-object]],
         and [Figure 5](#fig-attStructs).
+
+    :   {{AuthenticatorAttestationResponse/getTransports()}}
+    ::  This operation returns the value of {{AuthenticatorAttestationResponse/[[transports]]}}.
+
+    :   <dfn>\[[transports]]</dfn>
+    ::  This [=internal slot=] contains a sequence of one or more {{AuthenticatorTransport}} values. The first value is the transport used by the user agent to perform a {{PublicKeyCredential/[[Create]](origin, options, sameOriginWithAncestors)}} operation. Subsequent values are additional transports that the [=authenticator=] is believed to support. The sequence MUST NOT contain duplicate values.
```

Comment:
Proposed per [https://github.com/w3c/webauthn/pull/1050/files#r216992616](emlun's comment): 
> Subsequent values are additional transports that the [=authenticator=] is believed to support, and MUST be sorted lexicographically.

Reply:
Done.

Reply:
the requirement to sort ("and MUST be sorted lexicographically") doesn't seem to be reflected in the spec yet .

Reply:
> the requirement to sort ("and MUST be sorted lexicographically") doesn't seem to be reflected in the spec yet .

It doesn't have a `MUST`, but the list is defined that way each time. As that language is normative anyway, it'd be wrong to use ordering other than lexicographical. I think it's OK as-is, but feel free to object @rlin1, it'd be fine by me to use a `MUST` here, even if I don't think it's required.